### PR TITLE
tmux-devel: update devel port to the latest commit [5f662d]

### PR DESCRIPTION
**Pull request by tessus**
###### Description
update devel port to the latest commit [5f662d]

###### Tested on
macOS 10.11.6
Xcode 8.2.1

###### Verification 
Have you
- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?

- [x] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: sysutils/tmux/Portfile [hunk 1]
```diff
@@ -9,8 +9,8 @@ if {${subport} eq ${name}} {
     conflicts       tmux-devel
 }
 subport tmux-devel {
-    github.setup    tmux tmux 52869ed182482c26163799a7215139f4d81b6fca
-    version         20160929-[string range ${github.version} 0 6]
+    github.setup    tmux tmux 5f662d91db658abbcd46d7a678c0250754be35e3
+    version         20170406-[string range ${github.version} 0 6]
     revision        1
```
Comment (mojca) [inline sysutils/tmux/Portfile]: You would normally delete the revision.

Comment (tessus) [inline sysutils/tmux/Portfile]: ok, I can remove the revision.

Until now, the committers always left it in, thus I never thought this was a problem.

**Comment by tessus** [comment]
@mojca done

**Comment by tessus** [comment]
@mojca I've removed the revision as you requested

**Comment by mojca** [comment]
As I said, this already looked good to me (the thing about revision was mostly nitpicking), but didn't want to commit without having time to test.

Next time you should squash commits and force update the branch to avoid multiple commits.
I also added your github handle to the port as it wasn't obvious you are maintainer otherwise (but this is primarily our problem, not yours).

**Comment by mojca** [comment]
And I shortened description in the first line to make it fit into suggested length.

**Comment by tessus** [comment]
@mojca I read somewhere (I think it was in the commit guidelines that forced pushes are forbidden), otherwise I would have done so.

**Comment by tessus** [comment]
I'm really puzzled by the fact that a lot of information is missing in the documentation. Where is the documentation of the port groups? There's a section, but it only describes a few.

I couldn't find anything about the cxx11 port group, nor the other one you mentioned: qmake.

**Comment by tessus** [comment]
https://trac.macports.org/wiki/CommitMessages

You can correct any commit you have not pushed to GitHub by ​amending it or doing an ​interactive rebase. It's good practice to review local commits before pushing them.
**You cannot correct commits that you have already pushed. Comment on the commit's GitHub page instead. (Modifying your local history causes it to diverge from the public history. Publishing this alternate history would require force-pushing, which we ​disallow because it's ​bad practice.)** 

Can you please confirm that I am allowed to force push?

**Comment by mojca** [comment]
Forced pushes to the master branch are forbidden, yes, but you can (and should when that makes sense) force push to your own branch before it goes to master (which is what I just did to this PR).

Documentation for PortGroups is in a very suboptimal state, but we should urgently document at least cxx11. I usually look at the files themselves, for example `_resources/port1.0/group/cxx11-1.1.tcl` or `_resources/port1.0/group/cxx11-1.0.tcl`

**Comment by tessus** [comment]
Great, thanks for the info.

Yes, I usually always force push, if I have to change a PR, but the documentation of MacPorts is not always very clear. I really felt dumb, not doing it, but hey, I didn't want to screw up the MacPorts process.

Ok, I will look at the files directly. I was thinking about a variant for ansifilter that builds the qt gui. But in this case, a port group might help. something like qt4-devel. But since I couldn't find any documentation I haven't worked on it yet.

Oh, btw, I've fixed ansifilter's makefile upstream. Sent a PR and he he merged it already.
We now have CXXFLAGS and an EXTRA_LDFLAGS var to use.

**Comment by mojca** [comment]
There are PortGroups for Qt GUIs. You can write to the mailing list and ask for help *and* remind everyone to try to improve the documentation.

**Pull request merged**